Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt

```diff
@@ -91,7 +102,9 @@ class HomeFragment : Fragment(R.layout.fragment_home), AutoInject {
         }
 
         vm.showLoweredRiskLevelDialog.observe2(this) {
-            if (it) { showRiskLevelLoweredDialog() }
+            if (it) {
+                showRiskLevelLoweredDialog()
+            }
```

Comment:
```suggestion
 if (it) { showRiskLevelLoweredDialog() }
```

Probably your lint settings changed it. but makes sense to have it in single line

Reply:
You're right, it was the linter doing it's stuff. I agree, it should be a one liner.